doc/install.md

```diff
@@ -238,11 +238,15 @@ import pygmt
 pygmt.show_versions()
 
 fig = pygmt.Figure()
-fig.coast(region="g", frame=True, shorelines=1)
+fig.coast(projection="N15c", region="g", frame=True, land="tan", water="lightblue")
+fig.text(position="MC", text="PyGMT", font="80p,Helvetica-Bold,red@75")
 fig.show()
 ```
-
-If you see a global map with shorelines, then you're all set.
+![pygmt-get-started](https://github.com/GenericMappingTools/pygmt/assets/3974108/f7f51484-8640-4b58-ae5b-6c71e7150f7a)
+You should see a global map with land and water masses colored in tan and lightblue, respectively. On top, there
+should be the semi-transparent text "PyGMT". If the semi-transparency does not show up, there is probably an
+incompatibility between your GMT and Ghostscript versions. For details, please run `pygmt.show_versions()` and
+see [Not working transparency](#not-working-transparency).
```

Comment:
Please rewrap this sentence to no longer than 88 characters per line.

Reply:
Done in commit https://github.com/GenericMappingTools/pygmt/pull/3249/commits/cbb8cb73a14ed50cdb8cb99dbb649541ff1380f7.